interfaces/api/v1/workbench/llm_control.py

```
@@ -16,7 +16,7 @@
     LLMControlService,
 )
 from infrastructure.ai.provider_factory import LLMProviderFactory
-from infrastructure.ai.prompt_manager import get_prompt_manager
+# from infrastructure.ai.prompt_manager import get_prompt_manager  # 暂时注释，等待原仓库修复
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)